### ISSUE TITLE
Reply-to cannot be edited

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/quickFilters#107]
When we use the "reply-to" filter option, it adds the choice "Reply-To" (capitalized first letters) as a search term, however this term cannot be edited anymore, and no visible value is added by the assistant:

![image](https://user-images.githubusercontent.com/6314674/171383319-6e88137d-9d74-4a56-88c1-d5cdad44e94a.png)

There should be a text edit field to the right. Adding "reply-to" (all lowercase) as a custom field works, but it is not autofilled by the assistant.

**Comment by RealRaven2000** [comment]
Here is a fixed version:

[quickFilters-wx-5.5.2pre4.zip](https://github.com/RealRaven2000/quickFilters/files/8813536/quickFilters-wx-5.5.2pre4.zip)

This works in Thunderbird 91, but there is still a bigger issue in Thunderbird 101.b4 in that the line:

```javascript
messenger.WindowListener.registerWindow("chrome://messenger/content/FilterEditor.xhtml", 
  "chrome/content/scripts/qFi-filterEditor.js");
```

is loaded too late, so that the custom search terms cannot be patched.

**Comment by RealRaven2000** [comment]
For the issue with Thunderbird 102 I have raised issue #108

**Comment by RealRaven2000** [comment]
Latest version:

[quickFilters-wx-5.5.2pre19.zip](https://github.com/RealRaven2000/quickFilters/files/8823347/quickFilters-wx-5.5.2pre19.zip)

To install, download zip file and drag into Thunderbird Add-ons Manager (do not unpack contents)

**Comment by RealRaven2000** [comment]
Fixed in 5.5.4 - released today
